packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -39,7 +39,12 @@ const List<String> _kAvailablePlatforms = <String>[
 ];
 
 const String _kNoPlatformsErrorMessage = '''
-The plugin project was generated without specifying the `--platforms` flag, no new platforms are added.
+The plugin project does not seem to support any platform.
```

Comment:
nit: "seem" is kind of a weak statement. I would say something stronger like:

"The plugin project is not configured to support any platforms"

Reply:
I made it weak due to the "web" issue I mentioned below. I'm not sure what I prefer, do you think we should still make it a strong statement then mentioned the "web" issue too?